rke/k8s_rke_system_images.go

```diff
@@ -7450,6 +7450,49 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		"v1.23.6-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.3",
+			Kubernetes:                "rancher/hyperkube:v1.23.6-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.80",
+			NginxProxy:                "rancher/rke-tools:v0.1.80",
+			CertDownloader:            "rancher/rke-tools:v0.1.80",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.80",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-node-cache:1.21.1",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.21.1",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.21.1",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.5",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.22.0",
+			CalicoCNI:                 "rancher/mirrored-calico-cni:v3.22.0",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.22.0",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.22.0",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.22.0",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.22.0",
+			CanalCNI:                  "rancher/mirrored-calico-cni:v3.22.0",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.22.0",
+			CanalFlannel:              "rancher/mirrored-flannelcni-flannel:v0.17.0",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.22.0",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:5.1.1.0.1ae238a",
+			AciHostContainer:          "noiro/aci-containers-host:5.1.1.0.1ae238a",
+			AciOpflexContainer:        "noiro/opflex:5.1.1.0.1ae238a",
+			AciMcastContainer:         "noiro/opflex:5.1.1.0.1ae238a",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:5.1.1.0.1ae238a",
+			AciControllerContainer:    "noiro/aci-containers-controller:5.1.1.0.1ae238a",
+			AciGbpServerContainer:     "noiro/gbp-server:5.1.1.0.1ae238a",
+			AciOpflexServerContainer:  "noiro/opflex-server:5.1.1.0.1ae238a",
+			PodInfraContainer:         "rancher/mirrored-pause:3.6",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.2.0-rancher1",
+			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher1",
+			IngressWebhook:            "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.1.1",
+			MetricsServer:             "rancher/mirrored-metrics-server:v0.6.1",
+			CoreDNS:                   "rancher/mirrored-coredns-coredns:1.9.0",
```

Comment:
We don't have an rke template for coredns 1.9.0. Can you help me understand how we can use this? Also v1.8.3-rancher2 has the updated probe config https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.6/rke/templates/coredns_v1.8.3-rancher2.go so if we pull in 1.9.0 we may lose those updates and the coredns pod might be unhealthy

Reply:
The existing template named  `CoreDNSTemplateV183Rancher2` is used in this case. The rule is defined here: https://github.com/rancher/kontainer-driver-metadata/blob/491fce701b598ccc5db7811b007331169efa966f/rke/templates/templates.go#L146

Our templates are built based on the upstream one: https://github.com/coredns/deployment/blame/master/kubernetes/coredns.yaml.sed
and the upstream one does not contain any change except bumping the image tag. 
So I think we are safe to use  `CoreDNSTemplateV183Rancher2` for installing coredns 1.9.0

Reply:
Glad I checked. Sounds good to me